pkg/virt-controller/watch/ovm_test.go

```diff
@@ -232,11 +261,11 @@ func OfflineVirtualMachineFromVM(name string, vm *v1.VirtualMachine, started boo
 	return ovm
 }
 
-func DefaultOVM(started bool) (*v1.OfflineVirtualMachine, *v1.VirtualMachine) {
-	vm := v1.NewMinimalVM("testvm")
+func DefaultOVMWithNames(started bool, ovmName string, vmName string) (*v1.OfflineVirtualMachine, *v1.VirtualMachine) {
```

Comment:
I think we want to get away from using acronyms in function names.

Reply:
Sure, will get rid in the next (rebased again) upload